### ISSUE TITLE
Vaadin Gradle plugin 23.0.9 fails with Could not find com.vaadin:flow-plugin-base:23.0.7.

### ISSUE THREAD

**Issue by mvysny** [vaadin/flow#13723]
### Description of the bug

Vaadin gradle plugin, when applied, fails with

```
A problem occurred configuring root project 'karibu10-helloworld-application'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find com.vaadin:flow-plugin-base:23.0.7.
     Searched in the following locations:
       - https://maven.vaadin.com/vaadin-prereleases/com/vaadin/flow-plugin-base/23.0.7/flow-plugin-base-23.0.7.pom
       - https://plugins.gradle.org/m2/com/vaadin/flow-plugin-base/23.0.7/flow-plugin-base-23.0.7.pom
     Required by:
         project : > com.vaadin:com.vaadin.gradle.plugin:23.0.9 > com.vaadin:vaadin-gradle-plugin:23.0.9
```

Workaround is to add this to your build script:
```
buildscript {
    repositories {
        mavenCentral()
        maven { setUrl("https://plugins.gradle.org/m2/") }
    }
}
```

Gradle plugin 23.0.6 worked well without the buildscript workaround.

### Expected behavior

The Vaadin Gradle plugin should work without the buildscript{} workaround.

### Minimal reproducible example

git clone https://github.com/mvysny/karibu10-helloworld-application, uncomment the "buildscript {}" lines and try building the project.

### Versions

- Vaadin / Flow version: 23.0.9
- Java version: 11
- OS version: Ubuntu

**Comment by mvysny** [comment]
Vaadin 23.1.0 works well even without the `buildscript{}` workaround.

**Comment by mshabarov** [comment]
Closed because 23.1 has no such an issue anymore.
